Deployment/templates/continuous-testing.yml

```diff
@@ -0,0 +1,48 @@
+steps:
+    - checkout: self
+
+    - task: NodeTool@0
+      inputs:
+       versionSpec: '14.x'
+      displayName: 'Install Node.js'
+
+    - script: |
+       npm ci
+      displayName: 'npm restore'
+      workingDirectory: '$(Build.SourcesDirectory)/Tests'
+
+    - task: FileTransform@1
+      inputs:
+       folderPath: '$(Build.SourcesDirectory)\Tests\Configuration'
+       fileType: 'json'
+       targetFiles: 'appConfig.json'
+
+    - script: |
+       npm run test:auto
+      displayName: 'Run Functional Tests'
+      workingDirectory: '$(Build.SourcesDirectory)/Tests/AutoTests/FunctionalTest'
+
+    - task: PublishTestResults@2
+      condition: succeededOrFailed()
+      inputs:
+        testRunner: 'JUnit'
+        testResultsFiles: 'results.xml'
+        mergeTestResults: true
+        searchFolder:  '$(System.DefaultWorkingDirectory)\Tests'
+        testRunTitle: Dev Functional Test Results
```

Comment:
Could be QA 😮 
```
        testRunTitle: Dev Functional Test Results
```

Reply:
Environment name replace with environment variable.